lib/hex_web/email/sender.ex

```diff
@@ -0,0 +1,24 @@
+defmodule HexWeb.Email.Sender do
+  require Logger
+
+  def deliver_now(email) do
+    ses_rate = Application.get_env(:hex_web, :ses_rate) |> String.to_integer
+
+    email
+    |> recipients
+    |> recipient_chunks(ses_rate)
+    |> Enum.each(fn recipients ->
+      HexWeb.Throttle.wait(HexWeb.SESThrottle, length(recipients))
+      HexWeb.Mailer.deliver_now(email)
```

Comment:
the problem is that `email` in `HexWeb.Mailer.deliver_now(email)` still contains the original list of recipients. So I think the way to fix it, would be to overwrite the list of recipients to be a subset for each chunk: `email |> to(recipient_chunk) |> HexWeb.Mailer.deliver_now`. Not sure if this unblocks you, what's the exact problem you're having?